### ISSUE TITLE
As a SO/SE I would like to schedule articles to publish on future dates.

### ISSUE THREAD

**Issue by ekellagher** [CuBoulder/express#2443]
SO/ SE who have news items that they know will be published days or weeks in advance by the primary source (science magazine, granting org, etc.) cannot publish the news on their site until the primary source has released the info. Automatic post publishing scheduler would allow SO/ SE to create the content in advanced and publish their content in sync with the primary source release of info.
https://cuboulder.atlassian.net/browse/EXP-9733

**Comment by kreynen** [comment]
![screen shot 2018-12-18 at 11 47 46 am](https://user-images.githubusercontent.com/385234/50175881-29817280-02bb-11e9-83cb-ea96284906ef.png)
